### PR TITLE
Add HTA ADK1122H2 known issue

### PR DESCRIPTION
**Pull request by Ismawell**
Add Known issues about HTA applications and ADK for Windows 11, version 22H2

### CODE REVIEW

File: memdocs/configmgr/mdt/known-issues.md [hunk 1]
```diff
@@ -16,6 +16,44 @@ ms.collection: openauth
 
 This article provides details of any current known issues and limitations with the Microsoft Deployment Toolkit (MDT). It assumes familiarity with MDT version concepts, features, and capabilities.
 
+## HTA applications report Script error after upgrading to ADK for Windows 11, version 22H2
+After you updated your MDT boot image to [ADK for Windows 11, version 22H2](/windows-hardware/get-started/adk-install), HTA applications stop working and a message box is displayed: Script Error - An error has occured in the script on this page.
```
Comment (JohanFreelancer9) [inline memdocs/configmgr/mdt/known-issues.md:20]:
```suggestion
After you updated your MDT boot image to [ADK for Windows 11, version 22H2](/windows-hardware/get-started/adk-install), HTA applications stop working and a message box is displayed: Script Error - An error has occurred in the script on this page.
```

File: memdocs/configmgr/mdt/known-issues.md [hunk 2]
```diff
@@ -16,6 +16,44 @@ ms.collection: openauth
...
+
+HTA applications relies on MSHTML and starting Windows 11, version 22H2, the default legacy scripting engine was changed.
```
Comment (JohanFreelancer9) [inline memdocs/configmgr/mdt/known-issues.md:22]:
```suggestion
HTA applications rely on MSHTML and starting with Windows 11, version 22H2, the default legacy scripting engine was changed.
```

File: memdocs/configmgr/mdt/known-issues.md [hunk 3]
```diff
@@ -16,6 +16,44 @@ ms.collection: openauth
...
+
+To workaround the issue, you need to add the following registry value in WinPE:
```
Comment (JohanFreelancer9) [inline memdocs/configmgr/mdt/known-issues.md:24]:
```suggestion
To work around this issue you need to add the following registry value in WinPE: 
```

File: memdocs/configmgr/mdt/known-issues.md [hunk 4]
```diff
@@ -16,6 +16,44 @@ ms.collection: openauth
...
+## HTA applications report Script error after upgrading to ADK for Windows 11, version 22H2
+After you updated your MDT boot image to [ADK for Windows 11, version 22H2](/windows-hardware/get-started/adk-install), HTA applications stop working and a message box is displayed: Script Error - An error has occured in the script on this page.
+
+HTA applications relies on MSHTML and starting Windows 11, version 22H2, the default legacy scripting engine was changed.
+
+To workaround the issue, you need to add the following registry value in WinPE: 
+ ``` 
+ reg.exe add "HKLM\Software\Microsoft\Internet Explorer\Main" /t REG_DWORD /v JscriptReplacement /d 0 /f
+ ```
+To enable this change in MDT, we recommand you to backup the following file C:\Program Files\Microsoft Deployment Toolkit\Templates\Unattend_PE_x64.xml and to modify it like this:
```
Comment (JohanFreelancer9) [inline memdocs/configmgr/mdt/known-issues.md:28]:
```suggestion
To enable this change in MDT, we recommend that you back up the following file: `C:\Program Files\Microsoft Deployment Toolkit\Templates\Unattend_PE_x64.xml` and to modify it as follows:
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @Ismawell, kindly commit the suggested changes. Many thanks.

File: memdocs/configmgr/mdt/known-issues.md [hunk 5]
```diff
@@ -16,6 +16,44 @@ ms.collection: openauth
...
+## HTA applications report Script error after upgrading to ADK for Windows 11, version 22H2
+After you updated your MDT boot image to [ADK for Windows 11, version 22H2](/windows-hardware/get-started/adk-install), HTA applications stop working and a message box is displayed: Script Error - An error has occurred in the script on this page.
+
+HTA applications rely on MSHTML and starting with Windows 11, version 22H2, the default legacy scripting engine was changed.
+
+To work around this issue you need to add the following registry value in WinPE: 
+ ``` 
+ reg.exe add "HKLM\Software\Microsoft\Internet Explorer\Main" /t REG_DWORD /v JscriptReplacement /d 0 /f
+ ```
+To enable this change in MDT, we recommend that you back up the following file: `C:\Program Files\Microsoft Deployment Toolkit\Templates\Unattend_PE_x64.xml` and to modify it as follows:
```
Comment (JohanFreelancer9) [inline memdocs/configmgr/mdt/known-issues.md:28]:
```suggestion
To enable this change in MDT, we recommend that you back up the following file: `C:\Program Files\Microsoft Deployment Toolkit\Templates\Unattend_PE_x64.xml` and modify it as follows:
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@Ismawell, thank you for committing the suggested changes. Kindly also commit the minor additional suggested change. Many thanks.

**Review by BalaDelli — APPROVED** [review]
#sign-off

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**